### PR TITLE
Add/angular binding in chat

### PR DESCRIPTION
**Pull request by Dhruvi-Makwana**

### CODE REVIEW

File: Chatify/chat/static/js/chat.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                            user: "karan",
+                            profile: "http://127.0.0.1:8000/static/images/social_media.jpg",
+                            message: "hiii karan this is receiver karan",
+                            id: 1
+                        }]
+                    }
+            }
+
+        ]
+        console.log($scope.chatData)
```
Comment (yatishTT) [inline Chatify/chat/static/js/chat.js]: remove console

File: Chatify/chat/static/js/chat.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+
+    $scope.showChat = function(user) {
+            $scope.currentUser = user
+            console.log($scope.currentUser)
```
Comment (yatishTT) [inline Chatify/chat/static/js/chat.js]: remove concole

File: Chatify/chat/static/js/chat.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    };
+
+    $scope.sendChat = function(user){
+     var message = $scope.msgText;
+        alert($scope.msgText)
```
Comment (yatishTT) [inline Chatify/chat/static/js/chat.js]: remove logs console alert

File: Chatify/chat/static/js/chat.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        console.log(user)
+        console.log($scope.chatData[1].id)
+
+        var a = $scope.chatData[1].id
```
Comment (yatishTT) [inline Chatify/chat/static/js/chat.js]: it should be dynamic

File: Chatify/chat/static/js/chat.js [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+        if(user == a)
+        {
+            var text = $scope.msgText;
+            $scope.chatData[1].messages.sender.push(message);
```
Comment (yatishTT) [inline Chatify/chat/static/js/chat.js]: no static conditions should be there

File: Chatify/chat/static/js/register.js [hunk 6]
```diff
@@ -1,9 +1,10 @@
-$(document).on("submit", "#register_form", function(event) {
-        console.log("hii")
+
+ $(document).on("submit", "#register_form", function(event) {
```
Comment (yatishTT) [inline Chatify/chat/static/js/register.js:2]: use fucntions instead of event

**Review by yatishTT — CHANGES_REQUESTED** [review]

File: Chatify/chat/static/js/chat.js [hunk 7]
```diff
@@ -8,30 +8,79 @@ app.config(function($interpolateProvider) {
 
 app.controller('chatCtrl', function($scope, $http) {
 
-            $scope.UserData = []
-            const person = {
-                username: "Karan",
-                username1: "jaydip",
+$scope.currentUser = undefined;
+
+
```
Comment (HARSHKALAL) [inline Chatify/chat/static/js/chat.js:13]: why are you leaving 2 lines between two statemens

**Review by yatishTrootech — APPROVED** [review]

**Pull request merged**